content/en/profiler/enabling/dotnet.md

```diff
@@ -66,7 +66,7 @@ The following profiling features are available in the following minimum versions
 | Live heap profiling beta   | 2.22.0+                            | .NET 7+                                                                                  |
 | [Trace to Profiling integration][12]         | 2.30.0+                            | All supported runtime versions.                                                          |
 | [Endpoint Profiling][13]  | 2.15.0+                            | All supported runtime versions.                                                          |
-| Timeline                  | 2.30.0+                            | All supported runtime versions (except .NET 5+ required for garbage collection details). |
+| Timeline                  | 2.30.0+ (and 3.10.0+ for outgoing HTTP requests longer than 50 ms in beta)     | All supported runtime versions (except .NET 5+ required for garbage collection details and .NET 7+ required for outgoing HTTP requests). |
```

Comment:
```suggestion
| Timeline                  | 2.30.0+ (and 3.10.0+ for outgoing HTTP requests longer than 50 ms in Preview)     | All supported runtime versions (except .NET 5+ required for garbage collection details and .NET 7+ required for outgoing HTTP requests). |
```
Small update per Product lifecycle changes, however, ok to leave if your team has elected to keep "beta" wording for the libraries. (I see its used in other places on the page).